### DISCUSSION TITLE
What is the "Asset Discovery (Cache)" in cli-snapshot?

### DISCUSSION THREAD

**Discussion by cancan101** [percy/cli#509] [Support] [answered]
There is mention of the the [docs of command line args for cli-snapshot](https://github.com/percy/cli/tree/master/packages/cli-snapshot#readme) of an "asset discovery (cache)":
```
  -h, --allowed-hostname=allowed-hostname          allowed hostnames to capture in asset discovery
  -t, --network-idle-timeout=network-idle-timeout  asset discovery network idle timeout
  --debug                                          debug asset discovery and do not upload snapshots
  --disable-cache                                  disable asset discovery caches
```
but no documentation (as far as I can see) as to what this cache is and why you might need to disable it.

**Comment by Robdel12** [comment, marked as answer]
Hey @cancan101! `--disable-cache` will disable caching responses from the asset discovery browser. By default, we cache all 200 responses so we're not re-requesting the same resources for all snapshots (this can bring down local dev servers if you have a lot of snapshots).
